src/analyses/goto_check.cpp

```diff
@@ -1820,6 +1820,11 @@ void goto_checkt::goto_check(
 
         if(it->source_location.get_column()!=irep_idt())
           i_it->source_location.set_column(it->source_location.get_column());
+
+        if(mode==ID_java &&
+           it->source_location.get_java_bytecode_index()!=irep_idt())
```

Comment:
In principle yes, but I wanted to maintain the style used in the previous lines. I could potentially change them all in a different PR.